vendors/espressif/boards/ports/ble/nimble/iot_ble_hal_gatt_server.c

```diff
@@ -45,13 +45,17 @@
 
 #define BLE_GATT_DSC_CLT_CFG_UUID128 0xFB, 0x34, 0x9B, 0x5F, 0x80, 0x00, 0x00, 0x80, 0x00, 0x10, 0x00, 0x00, 0x02, 0x29, 0x00, 0x00
 
+#define BLE_MAX_ATTR_LEN 512
+
 static struct ble_gatt_svc_def espServices[ MAX_SERVICES + 1 ];
 static BTService_t * afrServices[ MAX_SERVICES ];
 static uint16_t serviceCnt = 0;
 static SemaphoreHandle_t xSem;
 static bool semInited;
 bool xSemLock = 0;
 uint16_t gattOffset = 0;
+static bool long_read_flag;
```

Comment:
The variables on lines 57&58 should be explicitly initialized, e.g. `static bool long_read_flag = false;`

Reply:
Okay, I did not explicitly initialized them as they are static variables, will update.